assets/app/scripts/controllers/create.js

```diff
@@ -33,6 +33,13 @@ angular.module('openshiftConsole')
       "" // "Other" category
     ];
 
+    // General tags shown at the top of the browse menu.
+    $scope.browseGeneral = ['instant-app', 'quickstart'];
+
+    // Specific technologies shown after the divider. Drop "instant-app", the
+    // first item, since it's general.
+    $scope.browseTechnologies = _.drop($scope.categoryTags, 1);
```

Comment:
hmm not sure if there is a lodash helper for it, but seems like it would be better to generically drop anything in the browseGeneral list from the categoryTags list